ti-linux-firmware/suite/bookworm/debian/control

```diff
@@ -0,0 +1,112 @@
+Source: ti-linux-firmware
+Section: kernel
+Priority: optional
+Maintainer: Sai Sree Kartheek Adivi <s-adivi@ti.com>
+Build-Depends:
+ debhelper-compat (= 12),
+ ca-certificates,
+Standards-Version: 4.5.0
+Rules-Requires-Root: no
+Vcs-Git: https://git.ti.com/git/processor-firmware/ti-linux-firmware.git
+Vcs-Browser: https://git.ti.com/cgit/processor-firmware/ti-linux-firmware
+Homepage: https://git.ti.com/cgit/processor-firmware/ti-linux-firmware
+
+Package: firmware-ti-ipc-am62
+Provides: firmware-ti-ipc
+Conflicts: firmware-ti-ipc
+Replaces: firmware-ti-ipc
+Depends: ${misc:Depends}
+Architecture: arm64
```

Comment:
Firmware blobs are not specific to the ARM, Arch should be "all".

Reply:
Sure. will update.